test/Microsoft.Azure.SignalR.Tests/ServiceLifetimeManagerFacts.cs

```diff
@@ -34,6 +37,9 @@ public class ServiceLifetimeManagerFacts
                 },
                 NullLogger<DefaultHubProtocolResolver>.Instance);
 
+        private static readonly IOptions<HubOptions> _hubOptions = Options.Create(new HubOptions() { SupportedProtocols = new List<string>() { "json", "messagepack" } });
+        private static readonly IOptions<HubOptions<TestHub>> _hubTOptions = Options.Create(new HubOptions<TestHub>() { SupportedProtocols = new List<string>() { "json", "messagepack" } });
```

Comment:
For a second I thought `_hubTOptions` related to strongly typed hubs. `_globalHubOptions` and `_localHubOptions` might make these tests slightly easier to read.